.circleci/config.yml.in

```diff
@@ -248,71 +226,119 @@ jobs:
 
   binary_win_conda:
     <<: *binary_common
-    executor:
-      name: win/default
-      shell: bash.exe
+    executor: windows-cpu
     steps:
       - checkout_merge
       - run:
           command: |
-            choco install miniconda3
-            (& "C:\tools\miniconda3\Scripts\conda.exe" "shell.powershell" "hook") | Out-String | Invoke-Expression
+            set -ex
+            if [[ "$CU_VERSION" == "cu92" ]]; then
+              export VC_YEAR=2017
+              export VSDEVCMD_ARGS="-vcvars_ver=14.11"
+              powershell packaging/windows/internal/vs2017_install.ps1
+            elif [[ "$CU_VERSION" == "cu100" ]]; then
+              export VC_YEAR=2017
+              export VSDEVCMD_ARGS=""
+              powershell packaging/windows/internal/vs2017_install.ps1
+            else
+              export VC_YEAR=2019
+              export VSDEVCMD_ARGS=""
+            fi
+            eval "$('/C/tools/miniconda3/Scripts/conda.exe' 'shell.bash' 'hook')"
             conda activate base
             conda install -yq conda-build "conda-package-handling!=1.5.0"
-            bash packaging/build_conda.sh
-          shell: powershell.exe
+            packaging/build_conda.sh
       - store_test_results:
           path: build_results/
 
   binary_win_conda_cuda:
     <<: *binary_common
-    executor: windows-gpu-prototype
+    executor: windows-gpu
     steps:
       - checkout_merge
       - run:
           command: |
-            choco install miniconda3
-            (& "C:\tools\miniconda3\Scripts\conda.exe" "shell.powershell" "hook") | Out-String | Invoke-Expression
+            set -ex
```

Comment:
This might be a follow up PR but it'd be nice if this logic was just captured in a re-usable command or a script so that we don't have to copy / paste it everywhere

Reply:
Sounds reasonable. Will fix.